java/lib/src/main/java/com/svix/MessageAttempt.java

```diff
@@ -70,9 +70,9 @@ public MessageAttemptOut get(final String msgId, final String appId, final Strin
 		}
 	}
 
-	public void resend(final String msgId, final String appId, final String endpointId) throws ApiException {
+	public void resend(final String msgId, final String appId, final String endpointId, final PostOptions options) throws ApiException {
```

Comment:
This breaks API. Every language that doesn't have a default will either need a new API added that's named differently, or in Java's case, the same API is fine, just need to create another one that's named the same.

Reply:
In java this is called method overloading: https://www.programiz.com/java-programming/method-overloading and is fine, python and go don't have this so it will have to be a function with a different name, perhaps `<X>WithOptions`

Reply:
Thanks, I was blanking. :)